src/core/command/command.manager.js

```diff
@@ -6,7 +6,17 @@ export class CommandManager {
 	invoke(commands) {
 		// First we need to get list of executable commands, cause execution of prev command can
 		// impact on canExecute of next command
-		this.apply(() => commands.forEach(cmd => cmd.execute()));
+		this.apply(() => {
+			const priorityCommands = Array.from(commands);
+			priorityCommands.sort((x,y) => y.priority - x.priority);
+
+			for (const cmd of priorityCommands) {
+				if (cmd.execute() === false) {
```

Comment:
don't you mean ```canExecute``` here? It looks more tenable by comment above and ```execute``` doesn't return boolean by d.ts file